### PR TITLE
[OSDOCS-12093]: Adding HCP content to 4.17 Welcome

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.17 only

Issue: https://issues.redhat.com/browse/OSDOCS-12093

Link to docs preview: https://84101--ocpdocs-pr.netlify.app/openshift-enterprise/latest/welcome/learn_more_about_openshift.html#self-managed-hcp

QE review:
- [x] QE has approved this change.

Additional information: Do not merge until the MCE 2.7 release on Nov. 5.

### CODE REVIEW

**Comment by lahinson** [comment]
@xenolinux - When you're back from PTO next week, PTAL at the Welcome page and let me know if you think I should highlight different topics or  any additional topics. Thank you!

**Review by xenolinux — APPROVED** [review]
Thanks for the welcome page PR.
The table looks great!

File: welcome/learn_more_about_openshift.adoc [hunk 1]
```diff
@@ -303,20 +303,40 @@ a|* xref:../nodes/scheduling/nodes-scheduler-taints-tolerations.adoc#nodes-sched
...
-// |
+| Deploying {hcp}
+a| * xref:../hosted_control_planes/hcp-deploy/hcp-deploy-virt.adoc#hcp-deploy-virt[Deploying {hcp} on {VirtProductName}] 
+* xref:../hosted_control_planes/hcp-deploy/hcp-deploy-aws.adoc#hcp-deploy-aws[Deploying {hcp} on {aws-short}] 
+* xref:../hosted_control_planes/hcp-deploy/hcp-deploy-bm.adoc#hcp-deploy-bm[Deploying {hcp} on bare metal] 
+* xref:../hosted_control_planes/hcp-deploy/hcp-deploy-non-bm.adoc#hcp-deploy-non-bm[Deploying {hcp} on non-bare-metal agent machines] 
+* xref:../hosted_control_planes/hcp-deploy/hcp-deploy-ibmz.adoc#hcp-deploy-ibmz[Deploying {hcp} on {ibm-z-title}] 
+* xref:../hosted_control_planes/hcp-deploy/hcp-deploy-ibmpower.adoc#hcp-deploy-ibmpower[Deploying {hcp} on {ibm-power-title}]
+
+| xref:../hosted_control_planes/hcp-disconnected/hcp-deploy-dc.adoc#hcp-deploy-dc[Deploying {hcp} in a disconnected environment]
```
Comment (mburke5678) [inline welcome/learn_more_about_openshift.adoc:333]: It's a bit confusing that this link actually goes to a topic called _Introduction to hosted control planes in a disconnected environment_. (That assembly does seem out of place; not sure where one would put it.)

**Comment by mburke5678** [comment]
@lahinson Just one comment. Otherwise LGTM

File: welcome/learn_more_about_openshift.adoc (hunk 1 above)
Comment (lahinson) [reply welcome/learn_more_about_openshift.adoc:333]: Good catch. I think I'll just remove the link from the phrase "Deploying hosted control planes in a disconnected environment".

**Pull request merged**